styles/settings-view.less

```diff
@@ -290,6 +290,10 @@
     margin: @section-padding 0;
   }
 
+  .sub-section.collapsed .package-container .row {
```

Comment:
I think this selector could be simplified to just `.sub-section.collapsed .package-container` – I'd also like to avoid referencing Bootstrap classes (like `.row`) since this package will stop using the Bootstrap grid at some point.